proposals/top-level-statements.md

```diff
@@ -81,15 +81,15 @@ static class Program
 }
 ```
 
-Note that the names "Program" and "Main" are used only for illustrations purposes, actual names used by
-compiler are implementation dependent and neither the type, nor the method can be referenced by name from
+Note that the names "Program" and "Main" are used only for illustration purposes, actual names used by the
+compiler are implementation-dependent and neither the type, nor the method can be referenced by name from
 source code.
 
 The method is designated as the entry point of the program. Explicitly declared methods that by convention 
-could be considered as an entry point candidates are ignored. A warning is reported when that happens. It is
```

Comment:
Please revert the `an` change. A better grammatical change would be to remove the `as`, but it reads fine as is.

Reply:
`entry point candidates` is plural, thus `an` is invalid in this context, and, as such, I removed `an`.

Reply:
Fair, I misread this.